pubspec.yaml

```diff
@@ -76,6 +76,7 @@ flutter:
     - assets/collections/
     - assets/images/icons/
     - assets/images/illustrations/
+    - assets/images/placeholders/
```

Comment:
The placeholder is an illustration, it could be placed directly in `/illustrations` instead creating a dedicated folder